src/app/pages/courses-page/components/courses-list/courses-list.component.html

```diff
@@ -1,6 +1,6 @@
-<ng-container *ngIf="filteredCourses.length; else noDataMessage">
+<ng-container *ngIf="coursesList.length; else noDataMessage">
   <ul>
-    <li *ngFor="let course of filteredCourses | orderBy: 'creationDate' : 'ASC'">
+    <li *ngFor="let course of coursesList">
```

Comment:
Why did you remove orderBy pipe? Update this pipe if it's necessary.

Reply:
Discussed and leave as it is for now.